test/log-join-concurrent.spec.js

```diff
@@ -14,42 +14,39 @@ const {
   stopIpfs
 } = require('orbit-db-test-utils')
 
-let ipfs, testIdentity
+let ipfs, testIdentity, identities
 
 Object.keys(testAPIs).forEach(IPFS => {
   describe('Log - Join Concurrent Entries (' + IPFS + ')', function () {
     this.timeout(config.timeout)
 
-    const { identityKeyFixtures, signingKeyFixtures, identityKeysPath, signingKeysPath } = config
+    const { identityKeyFixtures, identityKeysPath } = config
     const ipfsConfig = Object.assign({}, config.defaultIpfsConfig, {
       repo: config.defaultIpfsConfig.repo + '-log-join-concurrent' + new Date().getTime()
     })
 
     before(async () => {
       rmrf.sync(ipfsConfig.repo)
       rmrf.sync(identityKeysPath)
-      rmrf.sync(signingKeysPath)
       await fs.copy(identityKeyFixtures, identityKeysPath)
-      await fs.copy(signingKeyFixtures, signingKeysPath)
-      testIdentity = await IdentityProvider.createIdentity({ id: 'userA', identityKeysPath, signingKeysPath })
+      identities = new IdentityProvider()
+      testIdentity = await identities.createIdentity({ id: 'userA' })
       ipfs = await startIpfs(IPFS, ipfsConfig)
     })
 
     after(async () => {
       await stopIpfs(ipfs)
-      await testIdentity.provider.keystore.close()
-      await testIdentity.provider.signingKeystore.close()
+      await identities.keystore.close()
```

Comment:
Would it make sense to add `close()` directly to `identities` which then closes the internal keystore?